doc/src/vpicUnits.tex

```diff
@@ -37,41 +51,156 @@
+	\end{align*}
 
-	\noindent Note here that the system of units will depend on the electron density used to calculate $\omega_{\mathrm{pe}}$. A more practical alternative for problems involving lasers is to choose $l_{\mathrm{deck}}=c/\omega_0$ where $\omega_0$ is the laser frequency, since it is more common to need to change density than the laser frequency.
+	\noindent Note that there is no longer any dependence on temperature.
 
-	\section{Particle Weighting}
+	\subsubsection{Laser-Plasma Units}
 
-	To define a species, the user specifies the macroparticle mass, charge, and weighting factor. These must be consistent with the above unit system to ensure the desired behaviour. To understand this, the role of the particle weight factor needs to be understood. In particular, a particle's motion in the EM fields only has explicit dependence on the charge-mass ratio, and so the weighting factor does not appear in the particle push. Instead, it is used when the code accumulates current from the particles onto the grid. This means that the correct particle weighting is determined by
+	For laser-plasma interaction problems, the unit system can again be modified by replacing the time-scale with the inverse laser frequency, so that unit system no longer depends on the electron density used to calculate $\omega_{\mathrm{pe}}$. The deck constants are then
 
-	\begin{equation}
-		w = \frac{\rho_{\alpha}}{\rho_{\alpha,\mathrm{MP}}},
-	\end{equation}
+	\begin{align*}
+		l_{\mathrm{deck}} &= \frac{1}{k_0}, &
+		c_{\mathrm{deck}} &= 1, \\
+		\varepsilon_{0,\mathrm{deck}} &= 1, &
+		r_{e,\mathrm{deck}} &= -1,
+	\end{align*}
 
-	\noindent where $\rho_{\alpha}$ and $\rho_{\alpha,\mathrm{MP}}$ are the charge densities of the physical particles and macroparticles respectively. Users unfamiliar with VPIC might otherwise expect that $w=n_{\alpha}/n_{\alpha,\mathrm{MP}}$ or $w=q_{\alpha}/q_{\alpha,\mathrm{MP}}$.
+	\noindent where $k_0 \equiv \omega_0/c$ is the vacuum laser wavenumber. The conversion factors then become
 
-	\section{Particle Momenta}
+	\begin{align*}
+		L &= \frac{1}{k_0}, &
+		T &= \frac{1}{\omega_0}, \\
+		M &= \ncrit k_0^3m_e, &
+		Q &= \ncrit k_0^3e, \\
+		E &= \frac{m_e}{e}\omega_0 c, &
+		B &= \frac{m_e}{e}\omega_0.
+	\end{align*}
+
+	\noindent where $\ncrit$ is the critical density $\ncrit \equiv m_e\varepsilon_0\omega_0^2/e^2$ In this unit system the simulation cell size and time step should satisfy $\Delta x \ll 1$ and $\Delta t \ll 1$ in order for the field solver to accurately model the laser. It is also necessary to verify that the cell size resolves the Debye length, which is often a more restrictive condition. Additionally, the normalised electric (or magnetic) field is the parameter $a_0$. For $a_0 \ll 1$ this is the electron quiver velocity divided by the speed of light, while $a_0 \gtrsim 1$ indicates a relativistic field intensity which will lead to effects such as relativistically induced transparency and relativistic self-focusing.
```

Comment:
Electric field normalized to what?

Reply:
I mean that the electric field in this unit system is equal to a0. I can rephrase the sentence to make that clearer.